doc/release/platforms/README.sgi.rst

```diff
@@ -49,8 +55,7 @@ This can be done using the following settings:
 
```

Comment:
The performance note above seems to be fifo related and may not be applicable to qthreads.  I would guess that qthreads has not been used on an Altix, though, so we can't really claim it would only affect fifo.  The mention of pthreads, at least, seems confusing.